.gitignore

```diff
@@ -1 +1,2 @@
-target
+/target
+/.mvn/wrapper/*.jar
```

Comment:
`maven-wrapper.jar` gets downloaded here the first time you run the `mvnw` script, so it's not necessary to distribute it in the repo.